[DOCS] GX Cloud Architecture Supporting Content (#11439)
great-expectations/great_expectations: docs/docusaurus/docs/reference/learn/gx_in_your_data_pipeline/gx_in_your_data_pipeline_lp.md

---
sidebar_label: 'GX in your data pipeline'
title: 'GX in your data pipeline'
hide_title: true

> description: Learn how GX can be integrated into a data pipeline to manage and monitor data quality.

Reviewer: I worry that "how" implies a functional tutorial. I suggest saying "where" instead.

```suggestion
description: Learn where GX can be integrated into a data pipeline to manage and monitor data quality.
```

hide_feedback_survey: true
pagination_next: null
pagination_prev: null
---

import LinkCardGrid from '@site/src/components/LinkCardGrid';
import LinkCard from '@site/src/components/LinkCard';
import OverviewCard from '@site/src/components/OverviewCard';

<OverviewCard title={frontMatter.title}>

>   Learn how GX can be integrated into a data pipeline to manage and monitor data quality.

Reviewer: Thanks for changing "how" to "where" in the page description. Sorry that I didn't notice in the first round that this language was in two places

```suggestion
  Learn where GX can be integrated into a data pipeline to manage and monitor data quality.
```

</OverviewCard>

<LinkCardGrid>

>   <LinkCard topIcon label="Ingestion" description="Validate raw data before writing it to your data warehouse so that you can quarantine bad records and identify bugs in your source system." to="/reference/learn/gx_in_your_data_pipeline/ingestion" icon="/img/actions_icon.svg"/>

Reviewer: ideally, these cards should have distinct icons. Here's a suggestion for an icon already in the repo

![Image](https://github.com/user-attachments/assets/b72f800c-86ce-4f3c-beb3-4304a2fa9727)

```suggestion
  <LinkCard topIcon label="Ingestion" description="Validate raw data before writing it to your data warehouse so that you can quarantine bad records and identify bugs in your source system." to="/reference/learn/gx_in_your_data_pipeline/ingestion" icon="/img/install_icon.svg"/>
```

  <LinkCard topIcon label="Transformation" description="Check the results of transformations in your warehouse and condition pipeline steps based on validation success or failure." to="/reference/learn/gx_in_your_data_pipeline/transformation" icon="/img/actions_icon.svg"/>

>   <LinkCard topIcon label="Delivery" description="Ensure unexpected patterns reveal business insights rather than data quality issues." to="/reference/learn/gx_in_your_data_pipeline/delivery" icon="/img/actions_icon.svg"/>

Reviewer: ideally, these cards should have distinct icons. Here's a suggestion for an icon already in the repo

![Image](https://github.com/user-attachments/assets/b3be6102-19aa-4011-a2cd-9bd646ca4ccd)

```suggestion
  <LinkCard topIcon label="Delivery" description="Ensure unexpected patterns reveal business insights rather than data quality issues." to="/reference/learn/gx_in_your_data_pipeline/delivery" icon="/img/example_cases_icon.svg"/>
```
